NEWS

```diff
@@ -46,6 +46,7 @@ v0.8.21-head 2016-xx-xx  The Irssi team <staff@irssi.org>
 	        openssl dgst -sha256 -c | \
 	        tr a-z A-Z
 
+	+ Remove support for DANE validation of TLS certificates.
```

Comment:
Maybe add more stuff from the commit log to this line? (or maybe we should start linking to commits)